common/sagas/wallet.tsx

```diff
@@ -168,7 +168,8 @@ function* broadcastTx(action: BroadcastTxRequestedAction): SagaIterator {
           txHash={txHash}
           blockExplorer={network.blockExplorer}
         />,
-        0
+        0,
+        txHash
```

Comment:
Rather than refactor all notifications to have manually specified IDs, what if we generated random IDs for each of them? You could use `Date.now()` or `Math.random()`

Reply:
Using `Math.random()` would be better for unique Id's like react keys, I used txHash because it was unique and already a property on the object.